### DISCUSSION TITLE
I'm not sure if the preview works

### DISCUSSION THREAD

**Discussion by adampatterson** [Roboroads/laravel-tinker#377] [Q&A]
I'd love to have the output show in the preview.  I have a shortcut that opens and closes the sidebar but I'd love to have a kind of 50/50 split horizontally without the sidebar.

As I was thinking about that I saw the preview buttons like you'd see in MarkDown but I haven't noticed anything in there before.

![image](https://github.com/user-attachments/assets/e1c7b176-da8e-4537-a754-2b8f5950a1e6)

**Comment by Roboroads** [comment]
As top  my knopwledge, you can just drag or move the icon for tinker on the rigth side to the bottom bar to have the tinker output on the bottom

**Reply by adampatterson** [reply]
I'll double check, I know that I can move the sidebar to the bottom like the terminal output.  But I'd like it to exist in the code tab.

Also, I love them plugin :)
